### DISCUSSION TITLE
実機登録ができません！

### DISCUSSION THREAD

**Discussion by KOTARO1209** [lit-iphone-course/lit-iphone-course#136] [Question-original]
## (1) 実現したいこと
Xcodeと自分のiPhone10を繋ぎたい
## (2) 実現するために自分が試した内容とその結果
Calculatorのプロジェクトを使って試してみました
## (3) 調べた内容や資料
Jrに聞きながら、PCやiPhoneの再起動をおこなった
## (4) 原因だと思われそうな箇所(ソースコードをコピペして貼る)
![スクリーンショット 2023-02-12 14 30 05](https://user-images.githubusercontent.com/75009903/218295166-8ccf0f1f-767c-4809-8478-d6890f0f044a.png)
## (5) GitHubのリンク
https://github.com/KOTARO1209/Calculator

**Comment by txxxxc** [comment]
Notionにあるトラブルシューティングの資料に対処法が載っているので一旦これ見ながら進めてもらえるとありがたいです(多分手順が間違ってるとかじゃないので、気長に待つしかないかも、、、)
https://lifeistech.notion.site/c0573327ff6342c7adbea265c7ed5e80

**Reply by KOTARO1209** [reply]
すいませんおそらくXcodeのバージョンを上げる必要があり、そのためにPC自体のバージョンを上げる必要があります。
Xcode自体入っているので精一杯でして、Adobeアプリほとんど消してやっている状態なのでPCの容量が本当にギリギリな状態です。なんとかして容量空けてみますので、実機登録は後日に回しても良いでしょうか、、

**Reply by txxxxc** [reply]
>なんとかして容量空けてみますので、実機登録は後日に回しても良いでしょうか、、

了解です！Xcodeのバージョンアップが終わり次第教えてもらえると嬉しいです🙇‍♂️
